src/Pkg.jl

```diff
@@ -573,7 +573,8 @@ const precompile_script = """
     foo_hash = Pkg.Artifacts.create_artifact(dir -> touch(joinpath(dir, "foo")))
     Pkg.Artifacts.bind_artifact!("./Artifacts.toml", "foo", foo_hash)
     # Also create multiple platform-specific ones because that's a codepath we need precompiled
-    Pkg.Artifacts.bind_artifact!("./Artifacts.toml", "foo_plat", foo_hash; platform=platform_key_abi())
+    using Pkg.BinaryPlatforms
+    Pkg.Artifacts.bind_artifact!("./Artifacts.toml", "foo_plat", foo_hash; platform=Pkg.Artifacts.platform_key_abi())
```

Comment:
```suggestion
    Pkg.Artifacts.bind_artifact!("./Artifacts.toml", "foo_plat", foo_hash; platform=Pkg.BinaryPlatforms.platform_key_abi())
```

Reply:
Forgot to push the change where I removed the prefix over just `using BinaryPlatforms`.